my-app/src/components/Cocina.jsx

```diff
@@ -0,0 +1,179 @@
+        const docs = [];
+        querySnapshot.forEach((comanda) => {
+          docs.push({ ...comanda.data(), id: comanda.id });
+        });
+        console.log(docs);
+        setOrders(docs);
+      });
+  };
+
+  const updateOrder = (orderId, status) => {
+    db.doc(`/orders/${orderId}`)
+      .update({ status, preparedAt: new Date() })
+      .then(() => {
+        setOrders(
+          orders.map((order) =>
+            order.id === orderId
+              ? { ...order, status, preparedAt: new Date() }
+              : order
+          )
+        );
+      });
+  };
+
+  useEffect(() => {
+    getOrders();
+    //console.log(getOrders);
+  }, []);
+
+  console.log(orders);
+
+  const components = orders.map((order) => {
+    return (
+      <span>
+        {/* {order.status ? (
+          <Fragment key={order.id}>
+            {" "}
+            <table className="showComponents-verde">
+              <tr>
+                <th>Cliente:{order.cliente}</th>
+                <th>Numero de mesa:{order.nroMesa}</th>
+              </tr>
+              <tr>
+                {order.platillos.map((platillo, idx) => (
+                  <tr key={`${order.id}-${platillo.nameProduct}-${idx}`}>
+                    {platillo.nameProduct}
+                  </tr>
+                ))}
+              </tr>
+              <button className="button">Estado</button>
+              <br></br>
+              <br></br>
+            </table>
+          </Fragment>
+        ) : (
+          <Fragment key={order.id}>
+            {" "}
+            <table className="showComponents">
+              <tr>
+                <th>Cliente:{order.cliente}</th>
+                <th>Numero de mesa:{order.nroMesa}</th>
+                <th>
+                  {" "}
+                  <Fecha />{" "}
+                </th>
+              </tr>
+              <tr>
+                {order.platillos.map((platillo, idx) => (
+                  <tr key={`${order.id}-${platillo.nameProduct}-${idx}`}>
+                    {platillo.nameProduct}
+                  </tr>
+                ))}
+              </tr>
+              <button className="button" onClick={() => updateOrder(order.id)}>
+                Cambiar a preparada
+              </button>
+              <br></br>
+              <br></br>
+            </table>
+          </Fragment>
+        )} */}
+      </span>
+    );
+  });
+
+  //   const OrdersComponentes = ({ count }) => (
+  //     <div>
+  //       {count.map((a) => (
+  //         <table>
+  //           <tr>
+  //             <td>{a.cliente}</td>
+  //             <td>{a.numesa}</td>
+  //             <td>{a.order}</td>
+  //           </tr>
+  //         </table>
+  //       ))}
+  //     </div>
+  //   );
+
+  //   const components = count.map ((orders,index) => (
+  //     <OrdersComponentes count={count} key={`orders-${index}`} cliente={orders.cliente} numesa={orders.nroMesa} order={orders.platillos} />
+
+  //     )
+  //   )
```

Comment:
```suggestion
       </span>
```
si nada des esto sirve quítalo porque esta MAL dejar mucho código comentado